src/NServiceBus.Core/Hosting/StartupDiagnostics/WriteStartupDiagnostics.cs

```diff
@@ -23,7 +23,9 @@ protected override async Task OnStart(IMessageSession session)
 
             var duplicateNames = entries.GroupBy(item => item.Name)
                 .Where(group => group.Count() > 1)
+                .Select(group => group.Key)
```

Comment:
What led to this change? Seems fine, but also unrelated.

Reply:
We had a bug where we added a duplicate startup diagnostics section so I fixed that to figure out which one it was. I'd say its not worth the hazzle to pull that into a separe PR?